procedures/auxtel/observation_procedures/Parabolic_Focus_Sequence.ipynb

```diff
@@ -0,0 +1,619 @@
+{
```

Comment:
<p>consider merging these two cells. Something like</p><pre class="ql-syntax" spellcheck="false">test = await script.atcs.rem.ataos.evt_correctionEnabled.aget())

if not (test.m1 and test.atspectrograph and test.m1):
  await script.atcs.rem.ataos.cmd_enableCorrection.set_start(m1=True, hexapod=True, atspectrograph=True)
</pre><p> </p><p>you also might be able to put it in one line using a tertiary statement.... but i'm not sure.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/88/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Definitely better. Two cells merged. Replaced one of the test. m1 by test.m2. </p><p> </p>